src/libs/Navigation/linkTo.js

```diff
@@ -29,8 +66,8 @@ export default function linkTo(navigation, path, type) {
         if (action.payload.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR && getTopmostReportId(root.getState()) !== getTopmostReportId(state)) {
             action.type = 'PUSH';
 
-            // If this action is navigating to the RightModalNavigator and the last route on the root navigator is also RightModalNavigator
-            // then we want to replace the current RHP state with new one
+            // Type is up we deeplinked to the one of RHP flows and we want to replace the current screen with the previous one in the flow
```

Comment:
I _think_ this is what you were trying to say 
```suggestion
            // If the type is UP, we deeplinked into one of the RHP flows and we want to replace the current screen with the previous one in the flow
```